yt/visualization/plot_container.py

```diff
@@ -575,6 +582,14 @@ def save(self, name=None, suffix=".png", mpl_kwargs=None):
         names = []
         if mpl_kwargs is None:
             mpl_kwargs = {}
+        elif "format" in mpl_kwargs:
```

Comment:
I think this might be the best we can get, but format is not necessarily suffix, right?  Aren't there loose cases like `jpeg` and whatnot?